### ISSUE TITLE
Logging from Wasm running in browser

### ISSUE THREAD

**Issue by emilk** [rerun-io/rerun#2944]
Some users want to write Rust and/or C++, compile it to Wasm, run it in a browser, and use the Rerun logging SDK.

For maximum compatibility we will target single-threaded Wasm.

The logs would either be shown directly in the browser, or be sent somewhere else over WebSockets.

This is currently blocked on two things:
* Sending log events over WebSockets (from logging sdk -> server)
* Porting `re_sdk` to be wasm-web compatible

## Logging over WebSockets
`re_sdk` currently logs either to an `.rrd` file, or to a TCP stream. Adding a sink for WebSockets should be pretty trivial. We already use WebSockets for the server->viewer communication.

The server would probably need to listen for TCP and WebSocket connections on two different ports.

## Making `re_sdk` Wasm-ready
`re_sdk` currently make use of threads as a means of pipelining. Each log call is sent on a channel to a background thread for batching by `DataTableBatcher`, then sent over a channel to yet another thread where the sink sits (e.g. the TCP connection).

For Wasm we could do the batching directly upon each log call. That will slow down each log call though. We could also postpone it to later using `windw.setTimeout`. As each batch it produced it can be sent directly over WebSockets, or over a channel to a Rerun Viewer in the same web page.
